packages/java/engine-runtime/src/main/java/dev/hilla/internal/TaskGenerateHillaImpl.java

```diff
@@ -55,13 +53,13 @@ public void execute() throws ExecutionFailedException {
                 throw new ExecutionFailedException("Project directory not set");
             }
 
-            PluginConfiguration config = null;
+            EngineConfiguration config = null;
 
             if (buildDirectoryName != null) {
                 var buildDir = new File(projectDirectory, buildDirectoryName);
 
                 try {
-                    config = PluginConfiguration.load(buildDir);
+                    config = EngineConfiguration.load(buildDir);
```

Comment:
Probably, it would also be a good idea to improve my code here to not have nested try-catches 😅